File: Clase2/Actividad/Ejercicios en python/Ejercicio1.py
#Ingresar 10 valores por teclado, presentar la suma y promedio


valores = []

for i in range(10):
    valor = int(input(f"Introduce el valor {i+1}: "))
    valores.append(valor)
    
suma_total = sum(valores)
prom = suma_total/10

print(f"La suma total es de: ", suma_total)
print(f"El promedio de esta suma es: ", prom)

File: Clase2/Actividad/Ejercicios en C++/Ejercicio1.cpp
#include <iostream>
using namespace std;

int main() {
    int valores[10];  
    int suma = 0;     
    float promedio;   

    
    cout << "Ingresa tus valores:" << endl;
    for(int i = 0; i < 10; i++) {
        cout << " Ingresa el valor " << i + 1 << ": ";
        cin >> valores[i];
        suma += valores[i];  
    }

    promedio = suma / 10.0;

    cout << "La suma de los valores es: " << suma << endl;
    cout << "El promedio de los valores es: " << promedio << endl;

    return 0;
}